frontend/src/concepts/pipelines/content/configurePipelinesServer/const.ts

```diff
@@ -70,3 +70,6 @@ export const EMPTY_DATABASE_CONNECTION: EnvVariableDataEntry[] = [
     value: '',
   },
 ];
+
+export const storingDataHelpLink =
```

Comment:
nit: In the future, we name our const in this format

```tsx
const FOO_BAR = "foo bar"
```

Reply:
Seems like another case for linting, where there are many constants that don't use this format.

Reply:
i agree. I will merge this now and bring it up in quality